packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -2786,7 +2786,7 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
 
   /// Gets the line heights at the start and end of the selection for the given
   /// [EditableTextState].
-  _GlyphHeights _getGlyphHeights() {
+  ({double startGlyphHeight, double endGlyphHeight}) getGlyphHeights() {
```

Comment:
I just tried doing this and ended up changing it back. I was torn but my decision is because I worry about enshrining GlyphHeights as an official thing. Really the only reason I return both from the same method is because of performance reasons.